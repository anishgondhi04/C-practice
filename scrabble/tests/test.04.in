10
4
2 3 H hello
3 2 V hello
6 2 V howdy
5 6 H bye
0

