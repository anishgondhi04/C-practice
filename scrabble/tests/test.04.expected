..........
..........
...h..h...
..hello...
...l..w...
...l..d...
...o.bye..
..........
..........
..........
